installer/datafiles/Linux.data

```diff
@@ -144,13 +146,12 @@ fi
             ${{OMI_CONF_EDITOR}} httpsport -r 1270 < ${{OMI_CONF_FILE}} > ${{OMI_CONF_FILE}}_temp
             mv ${{OMI_CONF_FILE}}_temp ${{OMI_CONF_FILE}}
         fi
+        # Clean up logrotate
+        rm -f /etc/logrotate.d/scxagent
+        rm -f /etc/cron.d/scxagent
```

Comment:
Should not it be outside of conditon "#if DISABLE_PORT != true"

Reply:
will update it, thanks.

Reply:
Fixed, thanks.